test/cli_upgrade_test.go

```diff
@@ -29,3 +35,115 @@ func (suite *PouchUpgradeSuite) SetUpSuite(c *check.C) {
 func (suite *PouchUpgradeSuite) TeadDownTest(c *check.C) {
 	// TODO
 }
+
+// TestPouchUpgrade is to verify pouch upgrade command.
+func (suite *PouchUpgradeSuite) TestPouchUpgrade(c *check.C) {
+	name := "TestPouchUpgradeSuite"
+
+	command.PouchRun("run", "-d", "--name", name, busyboxImage).Assert(c, icmd.Success)
+
+	res := command.PouchRun("upgrade", "--name", name, busyboxImage)
+	c.Assert(res.Error, check.IsNil)
```

Comment:
I am wondering if we do not upgrade anything, should we return an error indicating `nothing to upgrade` and no status update happens in daemon side. @HusterWan WDYT